cmd/runtimetest/main.go

```diff
@@ -459,6 +459,122 @@ func validateMountsExist(spec *rspec.Spec) error {
 	return nil
 }
 
+func isParent(parent, child string) bool {
+	if parent == child {
+		return false
+	}
+
+	cparts := filepath.SplitList(child)
+	for i, part := range filepath.SplitList(parent) {
```

Comment:
>     I'm not sure what SplitList() here works for. Is a mount destination can be "pathA:pathB"?
> 
> |SplitList| splits a path into each component. So |"A/B"| becomes |[]string{"A", "B"}|. This code is checking if one path is lexically the parent of another. While this isn't bulletproof there isn't a nice way to deal with other cases like symlinks (though I could try using |ResolveSymlinks|).
> 

I think SplitList is not used for that.
In go document's example, it works like |"/a/b/c:/usr/bin"| becomes |[]string{"/a/b/c", "/usr/bin"}|.
And I tested, it really works like this.
Did I miss something?

>     But path in Windows generally looks like "C:\test". And mount path is not allowed nested in Windows.
> 
> I don't use Windows. If Windows doesn't support nested mounts, then this code is not useful for Windows.
> 

In config.md of runtime-spec, "For the Windows operating system, one mount destination MUST NOT be nested within another mount. (Ex: c:\foo and c:\foo\bar)."
So, I think we should make exception for Windows.